traffic_ops/traffic_ops_golang/api/api_route.go

```diff
@@ -0,0 +1,148 @@
+package api
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import "bytes"
+import "net/http"
+import "regexp"
+import "strconv"
+//import trops "github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang"
+import log "github.com/apache/trafficcontrol/lib/go-log"
+
+// todo - make the version string modular and accessible everywhere
+const ServerString = "Traffic Operations/3.0.0";
+const errorString = "Check the Traffic Ops log file(s) for details\n";
```

Comment:
Fair enough. I don't really have a better message for users, and I was reluctant to send it without an error message because it could make it hard to debug.